src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -1028,40 +1030,52 @@ def test_get_job_logs__job_id_list__one_ok_one_bad_one_fetch_fail(self):
         self.assertEqual(LOGS, msg["msg_type"])
 
         self.assertEqual(
-            set(list(msg["content"][JOB_COMPLETED].keys())),
-            set(["job_id", "batch_id", "first", "latest", "max_lines", "lines"]),
-        )
-        self.assertEqual(
+            msg["content"],
```

Comment:
with only 5 log lines to worry about, we can now test that the whole message is as we would expect it to be